components/HeroUnit.js

```diff
@@ -1,13 +1,26 @@
 import DownArrowIcon from '../resources/down-arrow-icon.svg'
 import NavigationBar from './NavigationBar'
+import ShuffleImagesByPixnodeGiveMeThatDamnTicket from 'react-pixnode-shuffle-images'
```

Comment:
Okay, I got it 😉!

Now shorten the name please 😛, e.g. `ShuffleImagesByPixnode`